docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-artifacts.md

```diff
@@ -166,15 +166,141 @@ Use the following instructions on your Linux machine, which this guide refers to
+    ```json
+    {
+      "image": [
+        {
+          "endpoint": "gcr.io",
+          "username": "_json_key",
+          "password": {
+            "type": "service_account",
+            "project_id": "spectro-images",
+            "private_key_id": "847c09190xxxxxxxxxxxxc4ebc",
+            "private_key": "-----BEGIN KEY-----MIIEvQIBADA ... -----Shortened for brevity",
+            "client_email": "xxx.iam.gserviceaccount.com",
+            "client_id": "115830xxxxxxx340453",
+            "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+            "token_uri": "https://oauth2.googleapis.com/token",
+            "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+            "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/spectro-images-viewer%40spectro-images.iam.gserviceaccount.com"
+          }
+        }
+      ]
+    }
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+    After creating the file, use the `-cred-file-path filepath` flag to include the credentials in the command.
+
+15. Issue the command below to create the content bundle. The `build` command uses the following flags:
 
-    | **Command Flag**        | **Value**                                                                                                                                           |
-    | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-    | `--api-key`             | Spectro Cloud API key                                                                                                                               |
-    | `--project-id`          | Palette project ID                                                                                                                                  |
-    | `--cluster-profile-ids` | Cluster profile IDs. If you want to include multiple cluster profiles in the content bundle, add multiple cluster profile IDs separated by a comma. |
-    | `--palette-endpoint`    | Palette API endpoint. The default Palette API endpoint is `api.spectrocloud.com`                                                                    |
-    | `--outfile`             | Path to write the final content bundle.                                                                                                             |
+    | **Command Flag**        | **Value**                                                                                                                                          |
+    | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | `--api-key`             | Spectro Cloud API key                                                                                                                              |
```

Comment:
```suggestion
    | `--api-key`             | Spectro Cloud API key.                                                                                                                              |
```

Reply:
Is this supposed to be Palette API key? Or is there a separate thing that we call Spectro Cloud API key?